support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -447,12 +463,16 @@ const customPrbStyles = css`
 type PrbType = 'APPLE_PAY' | 'GOOGLE_PAY' | 'PAY_NOW' | 'NONE';
 
 // ---- Component ----- //
-const PaymentRequestButton = (props: PropTypes) => {
+function PaymentRequestButton(props: PropTypes) {
 	const stripe = stripeJs.useStripe();
 	const [prbType, setPrbType] = useState<PrbType>('NONE');
 
 	function initialisePaymentRequest() {
-		const paymentRequestObject: PaymentRequestObject = stripe.paymentRequest({
+		if (!stripe) {
+			return;
+		}
```

Comment:
I guess this was never `null` before, but the stripe lib says it could be so good to finally check!